interface/main/holidays/import_holidays.php

```diff
@@ -0,0 +1,177 @@
+<?php
```

Comment:
Need a standard header. See here for an example:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#Page-Level_DocBlock